networking/tunnel/l2tp/basic/Makefile

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2019 Red Hat, Inc. All rights reserved. This copyrighted material
+# is made available to anyone wishing to use, modify, copy, or
+# redistribute it subject to the terms and conditions of the GNU General
+# Public License v.2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# Author: Jianlin Shi <jishi@redhat.com>
+
+TOPLEVEL_NAMESPACE=kernel
+
+PACKAGE_NAME=kernel
+
+RELATIVE_PATH=networking/tunnel/l2tp/basic
+
+# Version of the Test. Used with make tag.
+export TESTVERSION=1.0
+
+# The combined namespace of the test.
+export TEST=/$(TOPLEVEL_NAMESPACE)/$(RELATIVE_PATH)
+
+.PHONY: all install download clean
+
+# executables to be built should be added here, they will be generated on the system under test.
+BUILT_FILES= 
+
+# data files, .c files, scripts anything needed to either compile the test and/or run it.
+FILES=$(METADATA) runtest.sh Makefile
+
+run: $(FILES) build
+	./runtest.sh
+
+build: $(BUILT_FILES)
+	chmod a+x ./runtest.sh
+
+clean:
+	rm -f *~ *.rpm $(BUILT_FILES)
+
+# You may need to add other targets e.g. to build executables from source code
+# Add them here:
+
+
+# Include Common Makefile
+include /usr/share/rhts/lib/rhts-make.include
+
+# Generate the testinfo.desc here:
+$(METADATA): Makefile
+	@touch $(METADATA)
+	@echo "Owner:        Jianlin Shi<jishi@redhat.com>" > $(METADATA)
+	@echo "Name:         $(TEST)" >> $(METADATA)
+	@echo "Path:         $(TEST_DIR)"	>> $(METADATA)
+	@echo "License:      GPL" >> $(METADATA)
+	@echo "TestVersion:  $(TESTVERSION)"	>> $(METADATA)
+	@echo "Description:  l2tp basic">> $(METADATA)
+	@echo "TestTime:     30m" >> $(METADATA)
+	@echo "RunFor:       $(PACKAGE_NAME)" >> $(METADATA) 
+# add any other packages for which your test ought to run here
+	@echo "Requires:     $(PACKAGE_NAME)" >> $(METADATA)
+	@echo "Requires:     net-tools" >> $(METADATA)
+	@echo "RhtsRequires:    kernel-kernel-networking-common" >> $(METADATA)
```

Comment:
RhtsRequires will import internal RH libraries which won't work if running externally, these libraries have already been ported to tests-beaker github repo,  you can instead use RepoRequires like this example: https://github.com/CKI-project/tests-beaker/blob/master/networking/firewall/netfilter/target/Makefile#L84

```
@echo "RepoRequires: networking/common" >> $(METADATA)
@echo "RepoRequires: networking/tunnel/common" >> $(METADATA)
```